frontend/lib/norent/components/letter-counter.tsx

```diff
@@ -15,11 +16,13 @@ export const LetterCounter = () => {
           <p className="title is-spaced has-text-info">
             {NORENT_LETTERS_SENT_PRE_LAUNCH + session.norentLettersSent}
           </p>
-          <NorentLogo size="is-96x96" color="dark" />{" "}
-          <span className="subtitle">
-            letters sent by tenants across the USA
-          </span>
-          <p className="is-uppercase">Since April 2020</p>
+          <Trans>
+            <NorentLogo size="is-96x96" color="dark" />{" "}
+            <span className="subtitle">
+              letters sent by tenants across the USA
+            </span>
+            <p className="is-uppercase">Since April 2020</p>
+          </Trans>
```

Comment:
It's probably obvious to translators, but it might mayyybe be useful to add a `comment` attribute to the `<Trans>` explaining that a number appears before the translated text?  I dunno just a thought.